app/models/user.rb

```diff
@@ -83,6 +83,11 @@ def self.to_csv
   scope :unconfirmed, -> { where(confirmed_at: nil) }
   scope :locked_out, -> { where.not(locked_at: nil) }
 
+  # users created after public beta launch
+  scope :public_beta, -> { where(created_at: EarlyYearsFoundationRecovery::Application.public_beta_launch_date..Time.zone.now) }
```

Comment:
We could use `since_public_beta` which is self-documenting and would be understood by a BA.